### PR TITLE
Cleanup maps and data pages

### LINKED ISSUE #578: Broken markup on Custom Maps page
https://tnris.org/maps-and-data/custom-maps/ is all sorts of janky:
screen shot 2015-09-17 at 4 50 29 pm

### PR DESCRIPTION
**Pull request by wilsaj**
Misc cleanup of the formatting and language in the `maps-and-data` section of the site.

closes #578 

cc @christoball

### CODE REVIEW

File: content/markdown/maps-and-data/research-and-distribution.md [hunk 1]
```diff
@@ -3,15 +3,17 @@ template: page.html
 title: Research & Distribution Services (RDC)
 abstract: The RDC offers a variety of additional products, support and services from our in-house staff. We provide hands-on assistance and expertise.
 ---
+
 ### Some of our services include:
 
-* Statewide collection, duplication and distribution of USGS topographic, USFWS wetlands, and FEMA Flood Insurance Rate maps
-* Historic map and aerial imagery searches and/or reproductions
+* Statewide collection, duplication and distribution of maps from government partners such as USGS Topography, USFWS Wetlands, and the FEMA Flood Insurance Rate program
```
Comment (jseppi) [inline content/markdown/maps-and-data/research-and-distribution.md]: +serial comma after "duplication"

File: templates/maps-and-data.html [hunk 2]
```diff
@@ -16,11 +16,11 @@
             <div class="panel-body">
               <a href="{{m.link('order-data')}}" class="btn btn-success btn-block"><span class="glyphicon glyphicon-envelope"></span> Custom Data Order Form</a>
               <p>
-                This is a direct link to our Custom Data Order form to obtain customer service through the <a href="{{m.link('maps-and-data/research-and-distribution')}}"><abbr title="Research & Distribution">RDC</abbr></a>
+                This is a direct link to our order form 
               <p>
               <h4>Important Note</h4>
-              <p> <strong>Many datasets are available online and don't require custom ordering</strong></p>
-              <p>We recommend exploring our <a href="{{m.link('data-catalog')}}">data catalog</a> if you are unfamiliar with TNRIS data or require more information.</p>
+              <p> <strong>Many datasets are available online and don't require a custom order</strong></p>
```
Comment (jseppi) [inline templates/maps-and-data.html]: +period

File: templates/order-form.html [hunk 3]
```diff
@@ -17,8 +17,9 @@
 <div class="row">
   <div class="col-xs-12">
     <p>
-      This form is used to request pricing information and order data. The cost
-      will be based on the amount staff time and reproduction cost.
+      This form is used to initiate an order or request pricing information for
+      data and maps. The total price will be based on the amount staff time and
```
Comment (jseppi) [inline templates/order-form.html]: amount **of** staff time

File: content/markdown/maps-and-data/custom-maps.md [hunk 4]
```diff
@@ -14,29 +14,31 @@ TNRIS's trained cartographers and data specialists in the Research & Distributio
...
-Just fill out a custom map order form at (link) and include all available datasets you would like included in your map then TNRIS Staff will contact you with a quote.  You can even send us your own geospatial data to add to any custom map we are creating for you.  
+Just fill out our [order form](order-data) to start a custom map order. List all available
+datasets you would like to include in your map then TNRIS Staff will contact you
+with a quote. You can also send us your own geospatial data to add to any custom
+map we are creating for you.
 
 <div class="bs-callout bs-callout-danger"><strong>Important Note:</strong> All maps must be Texas and TNRIS data related. </div>
 
-## Map Examples
+## Examples of previous work
```
Comment (jseppi) [inline content/markdown/maps-and-data/custom-maps.md]: capitalization section headings

**Comment by wilsaj** [comment]
fixed - thanks!

**Comment by wilsaj** [comment]
rebased to #684 to fix merge conflict
closing this issue

**Pull request closed without merging**